src/components/SubmitReview.jsx

```diff
@@ -0,0 +1,76 @@
+import React, { Component } from 'react';
```

Comment:
Overall, the components look good with their mock data. Once the backend catches up, we can test it all out. You might want to change the file extensions to `.js` since AppMap has that extension.